ibex_demo_system_core.core

```diff
@@ -24,7 +24,7 @@ filesets:
 
   files_lint_verilator:
     files:
-      - lint/verilator_waiver.vlt: {file_type: vlt}
+      - vendor/lowrisc_ibex/lint/verilator_waiver.vlt: {file_type: vlt}
```

Comment:
I think this vlt file is already included by ibex core file

Reply:
Did you mean to add `demo_system_verilator_lint.vlt` here?

Reply:
Good point. Removed this file.